proto/config/edge_node_cluster.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package = "github.com/lf-edge/eve-api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+import "evecommon/acipherinfo.proto";
+
+message EdgeNodeCluster {
+  // cluster name, in case it has multiple cluster on the same site
+  string cluster_name = 1;
+
+  // cluster UUID
+  string cluster_id = 2;
+
+  // Cluster-Interface, for example "eth1"
+  string cluster_interface = 3;
+
+  // the 'cluster-prefix' IP address of the 'Cluster-Interface', 10.244.244.2/28
+  string cluster_ip_prefix = 4;
```

Comment:
Would this be used for the cluster interface instead of using [Network with IP config](https://github.com/lf-edge/eve-api/blob/main/proto/config/netconfig.proto#L19-L20)?

Reply:
That would be independent of the original network config on the cluster interface, if it is already there. It adds another prefix on the interface.

Reply:
Agreed.